src/core/hiwire.c

```diff
@@ -274,6 +280,22 @@ EM_JS_REF(JsRef, hiwire_int, (int val), {
   return Module.hiwire.new_value(val);
 });
 
+EM_JS_REF(JsRef, hiwire_int_from_hex, (const char* s), {
+  let result;
+  // clang-format off
+  if (HEAP8[s] === 45) {
```

Comment:
Could you add a comment why 45?

Reply:
My question was more how would one independently determine that 45 corresponds to the - sign.